### PR TITLE
Add in IsKinematic property to rigid body

### PR DESCRIPTION
**Pull request by tombuMS**
Referenced by issue #[344](https://github.com/microsoft/mixed-reality-extension-sdk/issues/344)

### CODE REVIEW

File: packages/sdk/src/types/runtime/rigidbody.ts [hunk 1]
```diff
@@ -23,6 +23,7 @@ export interface RigidBodyLike {
     detectCollisions: boolean;
     collisionDetectionMode: CollisionDetectionMode;
     useGravity: boolean;
+    isKinematic: boolean;
```
Comment (stevenvergenz) [inline packages/sdk/src/types/runtime/rigidbody.ts]: This whole interface needs documentation, but could you start here? Use Material for reference: [https://github.com/microsoft/mixed-reality-extension-sdk/blob/red/packages/sdk/src/types/runtime/assets/material.ts](https://github.com/microsoft/mixed-reality-extension-sdk/blob/red/packages/sdk/src/types/runtime/assets/material.ts)

Comment (tombuMS) [reply packages/sdk/src/types/runtime/rigidbody.ts]: Sure.  I am fine with beginning the documentation updates.

File: packages/sdk/src/types/runtime/rigidbody.ts [hunk 2]
```diff
@@ -15,38 +15,75 @@ import {
...
     mass: number;
+    /** Whether to detect collisions with this rigid body. */
     detectCollisions: boolean;
+    /** The collision detection mode to use with this rigid body.  @see CollisionDetectionMode for options. */
     collisionDetectionMode: CollisionDetectionMode;
+    /** Whether the rigid body is affected by gravity. */
     useGravity: boolean;
+    /** 
+     * Whether the rigid body is kinematic.  Note kinematic rigid bodies participate in collisions,
+     * but are not simulated by the rigid body.  This is usefull for objects that should collide with
```
Comment (eanders-MS) [inline packages/sdk/src/types/runtime/rigidbody.ts]: spelling: usefull -> useful

File: packages/sdk/src/types/runtime/rigidbody.ts [hunk 3]
```diff
@@ -59,6 +96,7 @@ export class RigidBody implements RigidBodyLike {
      */
 
     /**
+     * INTERNAL USE ONLY.
```
Comment (eanders-MS) [inline packages/sdk/src/types/runtime/rigidbody.ts]: Use @hidden instead to hide from doc generation

File: packages/sdk/src/types/runtime/rigidbody.ts (hunk 1 above)
Comment (eanders-MS) [reply packages/sdk/src/types/runtime/rigidbody.ts]: Thanks for adding documentation.

File: packages/sdk/src/types/runtime/rigidbody.ts (hunk 3 above)
Comment (tombuMS) [reply packages/sdk/src/types/runtime/rigidbody.ts]: Ok.  I thought that was what should be done, but I was following the example of the material.ts file like Steven had said.  I will just replace the internal use with @hidden then.

**Pull request merged**